Sources/SwiftGenKit/Parsers/Colors/ColorsParser.swift

```diff
@@ -52,7 +52,12 @@ public enum Colors {
     }
 
     public static var defaultFilter: String {
-      let extensions = Parser.subParsers.flatMap { $0.extensions }.sorted().joined(separator: "|")
+      let extensions = Parser.subParsers
+        .flatMap { subParser in
+          subParser.extensions.map { NSRegularExpression.escapedPattern(for: $0) }
+        }
+        .sorted()
+        .joined(separator: "|")
       return "[^/]\\.(?i:\(extensions))$"
```

Comment:
out of scope for that PR, but we definitively need to extract that logic into some helper method shared by all parsers, like `defaultFilter(for extensions: [String]) -> String` which will do all that manipulation, which would allow us to only write `defaultFilter(for: Parser.subParsers.flatMap(\.extensions))` in there, and similar calls in other parser's implementations. Or maybe that helper can directly take a `[Parser]` as input parameter even.

But that's something for a separate PR, just wanted to note to keep track.